docs/deploy.zh-CN.md

```diff
@@ -88,4 +76,83 @@ $ yarn build
 ```
 
 5. 如果第 4 步成功的话，那么构建后的文件在 `/dist` 目录下。
-6. 移动 `dist` 目录下的文件到 manager-api 的 `dist` 目录下，然后在浏览器中访问 `http://127.0.0.1:8080`，`8080` 是 manager-api 的默认监听端口。
+
+## 启动
+
+1. 根据您的部署环境，检查并修改 `api/conf/conf.json` 中的配置。
```

Comment:
This directory is too deep, can it be shortened later?

Reply:
yep, I'm refactoring the frontend and backend.